std/math.d

```diff
@@ -4066,11 +4066,11 @@ private:
         // ARM FPSCR is a 32bit register
         enum : int
         {
-            INEXACT_MASK   = 0x1000,
-            UNDERFLOW_MASK = 0x0800,
-            OVERFLOW_MASK  = 0x0400,
-            DIVBYZERO_MASK = 0x0200,
-            INVALID_MASK   = 0x0100
+            INEXACT_MASK   = 0x10,
+            UNDERFLOW_MASK = 0x08,
+            OVERFLOW_MASK  = 0x04,
+            DIVBYZERO_MASK = 0x02,
+            INVALID_MASK   = 0x01
```

Comment:
Yes, seems like those constants should be split up into architectures.
https://github.com/D-Programming-Language/druntime/blob/e66f03494a13c7ce13bbe4c260ce689682af865e/src/core/stdc/fenv.d#L250